### ISSUE TITLE
At 400% Zoom, While navigating using down arrow key, focus is not going to "searchForMoreText" prop value in the Tag Picker Suggestion pop up.

### ISSUE THREAD

**Issue by SreejaPampara** [microsoft/fluentui#19198]
### Environment Information

- **Package version(s)**: version 8.25.0
- **Browser and OS versions**: Chrome, Edge  - Windows 10

### Describe the issue:

When the control is zoomed to 400% and trying to navigate through items using down arrow key.
1. The "searchForMoreText" prop is not visible on the screen when the focus is on that button.
2. The active focused item is not visible in the screen 

### Please provide a reproduction of the issue in a codepen:
Here is the codepen link to repro the issue: 
https://codesandbox.io/s/competent-cannon-cfiz7
![image](https://user-images.githubusercontent.com/88161368/127604105-6d98c710-7b51-41de-a229-94d32bc67904.png)

Here the focus is on show more button but not visible on the screen while navigating through down arrow.

#### Actual behavior:
Show more button is not visible on the screen when  focus is on it while trying to  navigate through items using down arrow key.

#### Expected behavior:
Show more button should be visible on the screen when  focus is on it while trying to  navigate through items using down arrow key.

**Comment by gouttierre** [comment]
@SreejaPampara - thanks for filing this issue with us.

@smhigley -  Would you be able to confirm if this is a regression, or if this behavior is an issue? We can bring it up to the Shield team for prioritization

**Comment by smhigley** [comment]
Looks like this isn't confined to the Show More button -- the picker doesn't correctly calculate position for the out-of-view options when the height is too small, and they don't correctly scroll into view when they get visual focus.

We probably need to check the scrolling logic for all options at a small viewport height.
